addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-windows-instance.cfn.yml

```diff
@@ -155,11 +163,23 @@ Resources:
           FromPort: -1
           ToPort: -1
           CidrIp: !Ref AccessFromCIDRBlock
+        - !If
+          - AppStreamEnabled
+          - DestinationSecurityGroupId:
+              Fn::ImportValue: !Sub "${SolutionNamespace}-CfnEndpointSecurityGroup"
+            IpProtocol: '-1'
+          - !Ref "AWS::NoValue"
       SecurityGroupIngress:
         - IpProtocol: tcp
           FromPort: 3389
           ToPort: 3389
           CidrIp: !Ref AccessFromCIDRBlock
+        - !If
+          - AppStreamEnabled
+          - SourceSecurityGroupId:
+              Fn::ImportValue: !Sub "${SolutionNamespace}-SwbAppStreamSG"
+            IpProtocol: '-1'
+          - !Ref "AWS::NoValue"
```

Comment:
Can you add similar egress and ingress rules to RStudio as well. Similarly default security group needs to be added in NetworkInterfaces there.

Reply:
updated